docker/chromium/tests-syncer/Dockerfile

```diff
@@ -30,8 +30,9 @@ RUN apt-get update && \
 RUN apt-get update && \
```

Comment:
I think it's super confusing this is FROM gcr.io/clusterfuzz-images/base and not FROM gcr.io/clusterfuzz-images/chromium/base